src/resources/genomicsdb_export_config.proto

```diff
@@ -64,4 +64,6 @@ message ExportConfiguration {
   optional bool index_output_VCF = 13; 
   optional bool produce_GT_field = 14;
   optional bool produce_FILTER_field = 15;
+  optional bool sites_only_query = 16;
+  optional bool produce_GT_with_min_PL_value_for_spanning_deletions = 17;
```

Comment:
Is it safe to assume any default values for these fields? There are many defaults for the ImportConfiguration Protobuf , but none for the ExportConfiguration